site.yml

```diff
@@ -63,8 +63,8 @@ asciidoc:
     owncloud-changelog-url: https://owncloud.org/changelog/server/
     php-supported-versions-url: https://secure.php.net/supported-versions.php
     http-status-codes-base-url: https://developer.mozilla.org/en-US/docs/Web/Status
-    minimum-php-version: 7.0
-    minimum-php-version-shortcode: 70
+    minimum-php-version: '7.0'
+    minimum-php-version-short-code: 70
```

Comment:
I guess this is a "bug" in 10.4 and master branch also.

Reply:
It is. I just submitted two PRs to correct that mistake.

Reply:
PR #2455 is fixing that in master...